### PR TITLE
Feature/data explorer data documentation

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR adds the API documentation section to the data explorer. It would be nice to review the text as no all the info in the designs did apply.

image

### CODE REVIEW

File: app/javascript/app/components/data-explorer-content/api-documentation/api-documentation.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      ],
+      extra:
+        'All this parameters accept multiple values, separated by commas. Response is paginated. Pagination headers are in place.'
+    }
+  ],
+  'emission-pathways': [
+    {
+      title: 'GET /api/v1/data/emission_pathways/meta',
+      url: `${ESP_API}/data/emission_pathways/meta`,
+      description:
```
Comment (weberjavi) [inline app/javascript/app/components/data-explorer-content/api-documentation/api-documentation.js]: Maybe `description` (on meta) and `extra` (both on meta and content) properties could be stored in a variable as they have the same string in all the API calls. The same could be said about `start_year`, `end_year`, `sector`, `categories`, `countries`, `indicators` that are repeated in some of the calls.

File: app/javascript/app/components/data-explorer-content/api-documentation/api-documentation-component.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import infoIcon from 'assets/icons/link.svg';
```
Comment (weberjavi) [inline app/javascript/app/components/data-explorer-content/api-documentation/api-documentation-component.jsx]: Better rename this icon as `linkIcon`?

File: app/javascript/app/components/data-explorer-content/api-documentation/api-documentation.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        }
+      ],
+      extra:
+        'All this parameters accept multiple values, separated by commas. Response is paginated. Pagination headers are in place.'
+    }
+  ],
+  'emission-pathways': [
+    {
+      title: 'GET /api/v1/data/emission_pathways/meta',
+      url: `${ESP_API}/data/emission_pathways/meta`,
```
Comment (weberjavi) [inline app/javascript/app/components/data-explorer-content/api-documentation/api-documentation.js]: Emissions pathways meta link is triggering an error . 

image

**Comment by Bluesmile82** [comment]
Ready again!

**Comment by weberjavi** [comment]
@Bluesmile82 remember to merge develop before integrating this PR into `develop`

**Pull request merged**